common/display/vblankeventhandler.h

```diff
@@ -27,8 +27,6 @@
 
 #include "hwcthread.h"
 
-#define VBLANK_THRESHHOLD 2777777
```

Comment:
what does this threshold mean here?

Reply:
We ever think only when the different in vperiod is larger than this one, the 2.4 callback should be invoked.
For instance, when the period is changed from 90hz to 60hz